tests/mappings/mapping_q_eulerian_02.cc

```diff
@@ -105,8 +105,7 @@ void test ()
   // now the actual test
   DataOut<dim> data_out;
   data_out.attach_dof_handler(dof_handler);
-  std::vector<std::string> names (dim, "displacement");
-  data_out.add_data_vector(displacements,names);
+  data_out.add_data_vector(displacements, "displacement");
```

Comment:
If you do ``write_vtk`` of this, then visit and paraview crash when loading the file.

You either need to specify that it is a vector using DCI or only supply a single string (like I did) and ``add_data_vector`` will generate unique names (by appending _0, _1, _2). See the changes in the .output file.